src/openeo_pg_parser/translate.py

```diff
@@ -292,7 +292,7 @@ def translate_process_graph(pg_filepath, process_defs=None, parameters=None):
 
     # remove first layer of the process graph
     parameters = {} if parameters is None else parameters
-    if "parameters" in process_graph.keys():
+    if "parameters" in process_graph.keys() and process_graph.get("parameters"):
```

Comment:
Could you please change  `if "parameters" in process_graph.keys() and process_graph.get("parameters"):` to  `if process_graph.get("parameters"):`? `.get` returns `None` anyway if the key is not present in the dictionary.

Reply:
Good point I'll change this